pallets/avn/src/vote.rs

```diff
@@ -51,7 +50,7 @@ pub struct VotingSessionData<AccountId, BlockNumber> {
     /// The hard end time of this vote.
     pub end_of_voting_period: BlockNumber,
     /// The confirmations collected from the aye votes
-    pub confirmations: BoundedVec<ecdsa::Signature, MaximumValidatorsBound>,
+    // pub confirmations: BoundedVec<ecdsa::Signature, MaximumValidatorsBound>,
```

Comment:
```suggestion
```

Reply:
If we store this struct, we have to remember to do a state migration to remove this field